src/main/java/chess/model/piece/Piece.java

```diff
@@ -0,0 +1,48 @@
+package chess.model.piece;
+
+import java.util.Objects;
+
+import chess.vo.MoveType;
+import chess.vo.Path;
+import chess.vo.PieceColor;
+
+public abstract class Piece {
+
+    private final PieceColor pieceColor;
+
+    public Piece(PieceColor pieceColor) {
+        this.pieceColor = pieceColor;
+    }
+
+    public abstract boolean isMovable(Path path, MoveType moveType);
+
+    public String getEmblem() {
+        if (pieceColor == PieceColor.WHITE) {
+            return getConcreteEmblem().toLowerCase();
+        }
```

Comment:
```suggestion
        if (pieceColor.isWhite()) {
            return getConcreteEmblem().toLowerCase();
        }
```
꺼내서 비교하지말고, 메세지를 보내는형태로 변경해보세요~ 🙂

Reply:
역시 냅다 쓰던 `Enum`에 메세지를 보내면서 활용하면 더 사용하기 편하고 객체적이군요!!! 수정해보았습니다.🤸‍♂️